assignments/function-conversion.js

```diff
@@ -5,26 +5,41 @@
 // };
 // myFunction();
 
+let myFunction =()=>{};
+console.log(myFunction);
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {return param};
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+let add = (param1,param2) =>{return param1+param2};
```

Comment:
since you are using the fat arrow syntax AND you are only returning one line, you can remove the curly braces and the word 'return'. I would also recommend ALWAYS USING CONST to declare functions.

```const add = (param1,param2) => param1+param2;```